src/inet/UDPEndPointImplSockets.cpp

```diff
@@ -785,11 +781,22 @@ CHIP_ERROR UDPEndPointImplSockets::IPv4JoinLeaveMulticastGroupImpl(InterfaceId a
         interfaceAddr.s_addr = htonl(INADDR_ANY);
     }
 
+#ifdef SOCKET_ENABLE_MREQN
```

Comment:
It's now discouraged to use `#ifdef` in the SDK - we build with `-Wundef` to catch typos in the preprocessor directives. Therefore, this should be added to `InetConfig.h` and defined as 0 by default (and properly documented). Then this could be overridden in the platform's configuration like you did.

Reply:
Thanks, I didn't know if it was related to inet. How should I name it also ?

Reply:
You can follow the existing convention, something like `INET_CONFIG_UDP_SOCKET_MREQN` would probably work.